cookiecutter-django-app/hooks/post_gen_project.py

```diff
@@ -55,20 +51,12 @@ def move(src, dest):
 
 extra_context["placeholder_repo_name"] = python_placeholder_repo_name
 
-if import_template_from_github:
-    directory = "python-template"
```

Comment:
We were using the same `directory`, whether local or github, so we can combine this.